src/main/java/reposense/git/GitLog.java

```diff
@@ -43,7 +43,7 @@ public static String getWithFiles(RepoConfiguration config, Author author) {
 
         String command = "git log --no-merges -i ";
         command += GitUtil.convertToGitDateRangeArgs(config.getSinceDate(), config.getUntilDate());
-        command += " --pretty=format:" + PRETTY_FORMAT_STRING + " --stat";
+        command += " --pretty=format:" + PRETTY_FORMAT_STRING + " --numstat --shortstat";
```

Comment:
I changed from `--stat` to `--numstat --shorstat` is because:
- `--stat` only shows the sum of lines changed in file (instead of display addition and deletion separately`
- `--numstat --shortstat`'s files changed is easier to parse as the format is `<#addition>\t<#deletion>\t<file path>`